cmd/ip-masq-agent/ip-masq-agent_test.go

```diff
@@ -45,6 +45,36 @@ func NewFakeMasqDaemon() *MasqDaemon {
 	}
 }
 
+// Returns a MasqConfig with config values that are the same as the default values when the
+// noMasqueradeAllReservedRangesFlag is false.
+func NewMasqConfigNoReservedRanges() *MasqConfig {
```

Comment:
Can we just call `NewMasqConfig(false)` here? Seem a bit odd to maintain two set of CIDRs in codes and in test.

Reply:
So we need some way to make sure that whatever NewMasqConfig does is actually what we want - especially now that it is doing more than returning a constant. My thought was that either we create a new TestNewMasqConfig() case, or just test its output when called from other places. Since it's not actually a useful function external to this module, I decided on the latter approach.
I'm happy to be convinced either way though.